README.md

```diff
@@ -1,5 +1,103 @@
 # polymer-build
-A library of Gulp build tasks
 
-Note: This is an entirely different library from versions < 0.1.0 by wballard.
-The previous project is available at GitHub here: https://github.com/Custom-Elements/polymer-tools
+Polymer-build is a library for building Polymer projects. While the [Polymer CLI](https://github.com/Polymer/polymer-cli) is still the easiest way to build a project, it's customization options are limited. Polymer-build allows you to create a completely customized build pipeline, which can include additional streams and build tasks in any order. Use polymer-build over the CLI if you:
```

Comment:
Break after the first sentence and maybe add a sub-header after it like `## Relationship to Polymer CLI`.